mappings/net/minecraft/state/property/IntProperty.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS bwk net/minecraft/state/property/IntProperty
```

Comment:
It's a property of Integer objects, not so much primitive ints.

Reply:
The constructor takes unboxed ints and every use of the property handles values as unboxed ints. This is a cleaner name, and the verbosity of Integer over Int is redundant imo.